harbor-ui/src/routes/send.rs

```diff
@@ -34,7 +41,7 @@ pub fn send(harbor: &HarborWallet) -> Element<Message> {
         SvgIcon::UpRight,
         harbor.send_status == SendStatus::Sending,
     )
-    .on_press(Message::Send(harbor.send_dest_input_str.clone()));
+    .on_press_maybe(parse_send_destination(&harbor.send_dest_input_str).map(Message::Send));
```

Comment:
what happens if we fail to parse it?

Reply:
If `on_press_maybe()` is passed `None`, the button is disabled (see [here](https://docs.rs/iced/0.13.1/iced/widget/struct.Button.html#method.on_press_maybe))